bears/js/JSStandardBear.py

```diff
@@ -24,5 +24,16 @@ class JSStandardBear:
     CAN_FIX = {'Formatting'}
     SEE_MORE = 'https://standardjs.com/rules.html'
 
-    def create_arguments(self, filename, file, config_file):
-        return (filename, '--verbose')
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         parser: str = '',
+                         globals: str= '',
+                         plugin: str = ''):
+        args = ('--verbose', filename)
+        if parser != '':
+            args += ('--parser', parser)
+        if globals != '':
```

Comment:
why `globals` is in blue?

Reply:
I am not exactly sure. But it does refer to the value that gets passed to the globals parameter

Reply:
globals is built-in name and we are masking it... that is why